source/administration/devops-command-center.rst

```diff
@@ -3,199 +3,255 @@ Mattermost Incident Management
+  - Use the context menu of a post and select **Start incident**.
 
-You need to select a playbook and name your incident before you can select **Start Incident**. The incident description is optional. Only playbooks that you're a member of are listed in the **Playbook** drop-down menu.
+Starting an incident opens an interactive dialog. Provide an incident name and select a playbook to be used with the incident. Optionally provide a description to offer immediate context into the newly started incident. Only playbooks of which you are a member are listed in the **Playbook** drop-down menu.
 
-The creator of an incident is automatically added as the first member and becomes the commander. To change commanders, click the current commander's name in the RHS and select the new commander. Only members of the channel may be selected as commanders. To change commander to a user who is not in the channel, first add the user to the channel.
+The creator of an incident is automatically added as the first member and becomes the commander.
 
 Joining an incident
 ~~~~~~~~~~~~~~~~~~~
 
-When an incident has been started, it's added to the list of channels in the Mattermost team.
+Join an incident by joining the incident channel. If the incident channel is public, you may join the channel without permission. Search for and join the channel via **Browse Channels**.
 
-If an incident channel is private new participants can only be added to an incident channel by a channel member. If the incident is public, no invitation is needed - search for, and join, the channel via **Browse Channels** in Mattermost.
+If the incident channel is private, an existing member of the incident channel must invite you. System administrators may join private incident channels without permission.
+
+Working with stages and tasks
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any member of the incident channel may work with stages and tasks:
+* To mark a task as completed, select the unchecked checkbox next to the task. To undo this, clear the checkbox.
+* To advance to the next stage, select **Next Stage**. Advancing to the next stage without completing the tasks in the stage prompts you to confirm.
+* To assign a task to a member of the incident channel, select **No Assignee** (or the existing assignee's username), then select a user.
+* To view any description associated with a task, select the information icon to the right of the task name.
+* To execute a slash command associated with a task, select **Run** next to the listed slash command. Configured slash commands may be run as often as necessary.
+
+Changing commanders
+~~~~~~~~~~~~~~~~~~~
+
+To change commanders, use any one of the following:
+
+* Run the ``/incident commander @username`` slash command from the incident channel.
+* From the desktop or browser:
+  1. Select the **Incidents** icon in the channel header to open the right-hand sidebar.
+  2. Browse to the **Summary** tab.
```

Comment:
Does the user browse to a tab or select a tab? The answer to this will impact additional sections of this page.